docs/modules/operation/pages/service-assurance/monitors/MailTransportMonitor.adoc

```diff
@@ -167,18 +177,29 @@ Test that we can connect via IMAPS and open the OPENNMS folder.
 ----
 
 == Tests
```

Comment:
Thoughts on moving the `Tests` section to the top, just above the Monitor Facts table.  This would give more context to users before they read the details on the individual parameters.

Reply:
Agreed. That would be better.